src/screens/AdminPortal/ManageTag/editModal/EditUserTagModal.tsx

```
@@ -94,39 +95,49 @@ const EditUserTagModal: React.FC<InterfaceEditUserTagModalProps> = ({
+            onChange={(e): void => {
+              setNewTagName(e.target.value);
+            }}
+          />
+        </FormFieldGroup>
+      </Form>
+    </BaseModal>
   );
 };
 
-export default EditUserTagModal;
+export default EditUserTagModal;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)